wikipron/config.py

```diff
@@ -15,8 +15,9 @@
 
 # GH-49: Estonian and Slovak use @title = "wikipedia:{language} phonology".
 # GH-50: Korean has an extra "span" layer (for fonts) in //li[span[sup[a.
+# Some (but not all) Moksha pronunciations reside in //p[sup[a.
 _PRON_XPATH_SELECTOR_TEMPLATE = """
-//li[
+(//li|//p)[
```

Comment:
As far as I can tell, this shouldn't break anything. (I imagine this is why Kyle tagged me for code review -- let me know if there's anything else.)

If we confirm this is the case, plus an entry in `CHANGELOG.md`, then this PR is good to go.